Gemfile

```diff
@@ -43,3 +43,5 @@ gem "bridgetown-svg-inliner", "~> 2.0"
 gem "bridgetown-sitemap", "~> 2.0"
 
 gem "bridgetown-seo-tag", "~> 6.0"
+
+gem "bridgetown_internal_markdown_links", "~> 0.2.0"
```

Comment:
thats a simple, but nice addition. Could be nice if you can add it in a separate PR :pray: